src/transformStateWithClones.js

```diff
@@ -6,8 +7,53 @@
  *
  * @return {Object[]}
  */
+
+const addProperties = (actionExtraData, transformedState) => {
+  for (const key of Object.keys(actionExtraData)) {
+    const value = actionExtraData[key];
+
+    transformedState[key] = value;
+  }
+};
+
+const deleteCertainProperties = (actionKeysToRemove, transformedState) => {
```

Comment:
Instead of using 'actionKeysToRemove', consider renaming the parameter to 'keysToRemove' for better readability.